JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,13 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa. This demo demonstrates the following TabPanel features:      
+
+### Bind the TabPanel to Data       
+
+The [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property binds the TabPanel to data. To specify a custom template for all the TabPanel views, use the [itemTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate). To do the same for the tabs, use [itemTitleTemplate](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate).
+
+### Switch Between the Tabs     
+
+You can configure navigation between tabs. Specify the [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) to define whether to switch between views with the swipe gesture. You can also [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop) views and use the [animationEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled) property to specify whether to animate the change of the selected view.
```

Comment:
```suggestion
To switch between views with a swipe and animate the view change even on non-touch devices, set the [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) and [animationEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled) properties to **true**. You can also enable [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop) of views.

```

Reply:
Add that `loop` occurs when a right/left arrow is used?

Reply:
The swipeEnabled and animationEnabled properties do not depend on each other. You can enable swipe but disable animation on any device and vice versa.

>You can also enable [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop) of views.

We can say the same thing but shorter without the word 'enable'

Reply:
All three properties are of equal priority. What if use a list?
```
To configure navigation between tabs, use the following properties:

* [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled)           
Allows users to swipe views.

...
```

Or a "property - default value - description" table?

Reply:
I think a list would be ok, but Roman suggested on the previous review iteration to transform the list to a paragraph. I guess that the reason behind that was inconsistency two parts of the description whose first part should is a paragraph.

Reply:
No, I changed my mind, a list would be great. But it should after the selectedIndex paragraph so that the information about checkboxes go right after the list.